rust/agents/validator/src/validator.rs

```diff
@@ -289,4 +310,71 @@ impl Validator {
 }
 
 #[cfg(test)]
-mod test {}
+mod test {
+    use std::str::FromStr;
+
+    use ethers::{
+        signers::Wallet,
+        utils::{self},
+    };
+    use hyperlane_core::{Announcement, HyperlaneSigner, Signable, H256};
+    use hyperlane_ethereum::Signers;
+    use k256::ecdsa::SigningKey;
+
+    #[tokio::test]
+    async fn sign_manual() -> eyre::Result<()> {
```

Comment:
Seems like this test case maybe doesn't do anything anymore after we moved away from raw signers / bech32 validators and instead kept with the same ethereum-based announcement signatures?